seshdash/views.py

```diff
@@ -240,3 +252,35 @@ class UserList(generics.ListAPIView):
 class UserDetail(generics.RetrieveAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+
+def get_high_chart_data():
+     context_high_data = {}
```

Comment:
Please add some comments to this section. 

It's very difficult to understand what's going on.